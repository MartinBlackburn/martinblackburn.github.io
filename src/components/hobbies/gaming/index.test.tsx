// libraries
import { render } from "@testing-library/react";

// component under test
import GamingText from "./";

describe("GamingText component", () => {
    test("renders correctly", () => {
        const { asFragment } = render(<GamingText />);

        expect(asFragment()).toMatchSnapshot();
    });
});
