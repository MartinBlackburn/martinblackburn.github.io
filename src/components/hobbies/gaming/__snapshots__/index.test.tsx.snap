// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GamingText component renders correctly 1`] = `
<DocumentFragment>
  <p>
    Competition, puzzles, and the ability to transport yourself to an entirely new world and/or place in time. What's not to love about a little escape from reality, it's even better when you're with a few friends, and maybe even a drink or two.
  </p>
  <p>
    Currently we are playing through 
    <a
      href="https://borderlands.com/"
    >
      Borderlands 3
    </a>
    , god bless that little 
    <a
      href="https://en.wikipedia.org/wiki/Claptrap"
    >
      Claptrap
    </a>
    .
  </p>
</DocumentFragment>
`;
